genconfig.py

```diff
@@ -36,12 +36,12 @@ def get_servers_from_nodetool_status(filename):
 
 def dump_yaml(directory, filename, servers, cluster):
     try:
```

Comment:
you don't need "try-except" here.
(https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir)

Reply:
See example in my previous comment